tests/lit/single/conversion/to_string.st

```diff
@@ -0,0 +1,30 @@
+// RUN: (%COMPILE %s && %RUN) | %CHECK %s
+FUNCTION main : DINT
+    VAR
+        myChar : CHAR;
+        myString : STRING[1];
+    END_VAR
+
+    // TODO: There's a bug, printf('%s$N', TO_CHAR(STRING#'foo')); returns a false positive
```

Comment:
If this happens to be a segfault, you should be able to resolve it by changing `%s` to `%c?`

Reply:
Not a segfault, but already discussed in daily